jina/flow/base.py

```diff
@@ -974,27 +974,17 @@ def identity(self, value: str):
     # for backward support
     join = needs
 
-    def dump(self, pod_name, path: str, shards: int, timeout=-1):
-        """Emit a Dump request to a specific Pod
-
-        :param shards: the nr of shards in the dump
-        :param path: the path to which to dump
-        :param pod_name: the name of the pod
-        :param timeout: time to wait (seconds)
-        """
-        pod = self._pod_nodes[pod_name]
-        pod.dump(path, shards, timeout)
```

Comment:
While I agree, that in general this should be done via a separate client, do we really want to cut the ability to do it via the Flow object?